src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2ConnectionTests.cs

```diff
@@ -1727,7 +1727,7 @@ public async Task AbortConnectionAfterTooManyEnhanceYourCalms()
         await WaitForConnectionErrorAsync<Http2ConnectionErrorException>(
             ignoreNonGoAwayFrames: true,
             expectedLastStreamId: int.MaxValue,
-            expectedErrorCode: Http2ErrorCode.INTERNAL_ERROR,
+            expectedErrorCode: Http2ErrorCode.ENHANCE_YOUR_CALM,
```

Comment:
@amcasey I added a `HttpConnection.Abort` overload that has an `Http2ErrorCode` argument. When there are excessive resets I made the passed in value `ENHANCE_YOUR_CALM` instead of `INTERNAL_ERROR`. That seems a more appropriate error.

Did you choose `INTERNAL_ERROR` for a reason here, or is it returned because that is what abort already sent?

Reply:
This was a hotfix, so I suspect I was making the smallest change possible, but I no longer recall.